### PR TITLE
EndoStation

### LINKED ISSUE #2851: Zulu Station
I need feedback for the station, personally I think I will redo sec cause its just shit
but all the mappers and coders are busy so cant provide feedback so I must pass it over to you guys
please tear into me like we did to kilo station

a few things to note
**its a station for LRP least it was meant to be, same time I hope MRP get some use out of it
im not trying to reinvent the station and at same time not trying to make box extended
### it uses manual APCS
donuts SM heavily inspired Zulu SM but it adheres to beestations building standards (fireshutters and firedoors)**

Zulustation

### PR DESCRIPTION
**Pull request by LastCrusader105**
## About The Pull Request
Adds EndoStation a new map for beestation

credit to @qwertyquerty for the name

## Why It's Good For The Game
Adds a new station 
for you MRPersor HRPers
**what is EndoStation how was it made etc etc?**
EndoStation was made in a joint effort between Dr.Octagon and Nanotrasen after he convinced them being a plasma man he knew all about plasma and would make them both rich, then rumours started about Octagons weird obsession with gene splicing and creating monsters, and the station started to lose funding, Octagon is believed dead

hopefully you get the feel of this once vibrant station has lost some of its glamour and looks bit ran down, science was first to be underfunded orginally meant to show off Nanotrasen power and reach, least thats what Octagon told them, Its crew entertainment sector is reached soon as you get off the shuttle each department has a shuttle, one of Nanotrasens failed visions (future proofing map for away missions like sciences)

## Changelog
:cl:
add: new station EndoStation
add: more pods on centcoom, a air alarm for plasma rooms and a new pet a gronola Jammor IX, 2 new areas: bridge checkpoints
imageadd: added some area images 2 bridge checkpoints and one for AFT solar control
code: added, plumbing and morgue to mail system, adds new solar control area, adds a plasma alarm (good for plasmeme rooms)
/:cl:
 

zulu

### CODE REVIEW

**Comment by LastCrusader105** [comment]
more info can be found at https://github.com/BeeStation/BeeStation-Hornet/issues/2851
but for the lazy
files too big oof
heres the last one
ZuluStation

forgot to add spawn points one min
and fixed the area issue near perma, added near space on all lattices I missed

**Comment by PowerfulBacon** [comment]
Oh damn that was fast

File: code/modules/atmospherics/machinery/airalarm.dm [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
 /obj/machinery/airalarm
 	name = "air alarm"
-	desc = "A machine that monitors atmosphere levels and alerts if the area is dangerous."
+	desc = "A machine that monitors atmosphere levels. Goes off if the area is dangerous."
```
Comment (PowerfulBacon) [inline code/modules/atmospherics/machinery/airalarm.dm]: Old description sounded better

File: code/modules/atmospherics/machinery/airalarm.dm [hunk 2]
```diff
@@ -159,7 +178,7 @@
 
 /obj/machinery/airalarm/all_access
 	name = "all-access air alarm"
-	desc = "This particular atmospherics control unit appears to have no access restrictions."
+	desc = "This particular atmos control unit appears to have no access restrictions."
```
Comment (PowerfulBacon) [inline code/modules/atmospherics/machinery/airalarm.dm]: Old description sounded better again

File: code/modules/mob/living/simple_animal/friendly/gondola.dm [hunk 3]
```diff
@@ -63,3 +63,11 @@
 #undef GONDOLA_COLOR
 #undef GONDOLA_MOUSTACHE
 #undef GONDOLA_EYES
+
+
+/mob/living/simple_animal/pet/gondola/jammor_ix
+	name = "Jammor IX"
+	desc = "Jammor IX,one of Octagons weird experiments,donated by Nanotrasen after a failed revolt."
```
Comment (PowerfulBacon) [inline code/modules/mob/living/simple_animal/friendly/gondola.dm]:
```suggestion
	desc = "One of Octagon's weird experiments, donated by Nanotrasen after a failed revolt."
```

**Review by PowerfulBacon — COMMENTED** [review]
WIll look at maps in a bit

**Comment by PowerfulBacon** [comment]
Is this still WIP? I see no wires on the map

**Comment by PowerfulBacon** [comment]
Weird corner walls thing here
image

**Comment by PowerfulBacon** [comment]
Why is this wall reinforced?
image

**Comment by PowerfulBacon** [comment]
2 walls randomly not reinforced here 
image

**Comment by LastCrusader105** [comment]
the walls was a weakness on purpose

**Comment by PowerfulBacon** [comment]
I'd recommend trying to avoid these double thick walls, since they can look bad in game.
Also the floor pattern is a little messed up and this room seems kind of empty (maybe a few crates or something lying around).
image
image

**Comment by AnjelicaChaykovsky** [comment]
It's hard to see on photo but it looks like you copy pasted most rooms from other stations ?

**Comment by LastCrusader105** [comment]
sorted the wall and floor issue out

**Comment by LastCrusader105** [comment]
> It's hard to see on photo but it looks like you copy pasted most rooms from other stations ?

non of it was copied and pasted but yes most are made by hand copies of departments

**Comment by LastCrusader105** [comment]
sorted bridge

**Comment by LastCrusader105** [comment]
image

image

File: code/modules/atmospherics/machinery/airalarm.dm (hunk 1 above)
Comment (LastCrusader105) [reply code/modules/atmospherics/machinery/airalarm.dm]: that's what it is standard, I didn't change it

File: code/modules/atmospherics/machinery/airalarm.dm (hunk 2 above)
Comment (LastCrusader105) [reply code/modules/atmospherics/machinery/airalarm.dm]: i didnt change it

**Review by LastCrusader105 — COMMENTED** [review]
no idea what this is

**Comment by LastCrusader105** [comment]
i will check back in a few hours and do a mass fix,but heres the latest fixes

**Comment by PowerfulBacon** [comment]
The space around the station is quite bland, check out the other maps and add some things like lattices and reinforced walls + grilles

**Comment by PowerfulBacon** [comment]
The solar panels should have a small amount of wires to get started (also the station itself needs wires)

**Comment by PowerfulBacon** [comment]
These showers and borg toilet seem a little strangely placed (just connected to a main hall).
image
Also the floor sprite for the 80s floor is pretty hideous

**Comment by PowerfulBacon** [comment]
Diagonal wall thing again
image

**Comment by PowerfulBacon** [comment]
Armoury needs an external motion sensing camera (unless the HOS one is)
image

**Comment by PowerfulBacon** [comment]
This cremator is very strangely placed, and this long 1 wide hallway seems like it would be better of as maints
image

Also the chaplain has a bed in the middle of the empty 3x3 room, would be nice if it was spiced up a bit with dressers and what not

**Comment by PowerfulBacon** [comment]
Security area with normal easilly breachable walls and some double walls in maints
image
Also this hallway seems very weird since it extends slightly into maints

**Comment by PowerfulBacon** [comment]
These floor tiles
image

**Comment by PowerfulBacon** [comment]
image

**Comment by PowerfulBacon** [comment]
These floor decals are even worse
image

**Comment by PowerfulBacon** [comment]
Strange floor here
image

**Comment by PowerfulBacon** [comment]
Massive, empty room
image

**Comment by PowerfulBacon** [comment]
Floating APC
image
Also empty table, consider putting paper_bins down

**Comment by PowerfulBacon** [comment]
I know it's a maint hall, but the floor decals are very weird
image

**Comment by PowerfulBacon** [comment]
Evac crab?
image
Also I'd add some decals to evac, it seems pretty bland

**Comment by PowerfulBacon** [comment]
In general the map needs more decals in the subrooms, there are many rooms that are just large, empty spaces with nothing in them and no detail

**Comment by LastCrusader105** [comment]
this is why I did a pull, my idea one got no traffic and I need feedback

issue one, the 80 floor stays cause its ugly, as for shower room what do you think should go there?
issue 2 is solved
issue 3 yes its a alert camera forget the name but I add a second one anyway
issue 4 yeah originally it was meant as a corridor for Chaplin to hear prisoners confessions
I will rework the whole area and add a interrogation/torture room the incinerator is there so "mourners" can see their love one be cremated
issue 5 solved
issue 6 solved
issue 7 is that not resprited stun gun? if not it will be changed
issue 8 they are boxstations decals legit
issue 9 apc sorted and I will add some
issue 10 thank you I will solve
issue 11 yes evac crab

this is why I wanted feedback I have been round the station maybe 10 times past 2 days and I was just getting blind to these things, I need the community to look using fresh eyes
also whats travis issue?

**Comment by PowerfulBacon** [comment]
```
_maps/map_files/ZuluStation/ZuluStation.dmm:10194:error: bad type or variable: /datum/gas/o2

_maps/map_files/ZuluStation/ZuluStation.dmm:10194:error: expected )

Failed to load map file /home/travis/build/BeeStation/BeeStation-Hornet/_maps/map_files/ZuluStation/ZuluStation.dmm.
```

**Comment by LastCrusader105** [comment]
ive no idea what that means I dont code

I assume its a double pipe or something i will have a gander

**Comment by PowerfulBacon** [comment]
It's in the map file. Might be one of the miners.

**Comment by PowerfulBacon** [comment]
Also there probably needs to be windows here (ill continue looking tomorrow probably)
image

**Comment by PowerfulBacon** [comment]
These are not good engineer spawns, they spawn without mesons looking into the supermatter crystal
image

**Comment by LastCrusader105** [comment]
so we dont have too many screenshots easy ones (like cameras etc) I will just say they are sorted and your have to either take my word or check yourself
issue 4
issue4
issue 5 added cameras
issue 6 changed for white floors
issue 7 deleted added a empty stun baton
issue 8 think i just have standard med floors for that
issue 9 changed direction to 1
issue 10 made it line the corridor and fixed the pipe issue we didnt see
issue10
issue 11 solved he stays maybe I will name him

**Comment by LastCrusader105** [comment]
issue 12 while designing the AI and after speaking to my Ai mains, they liked the idea of using space as a weapon, i can fix it if you want but id call it a feature
issue 13 fixed oh that reminds me got to put sm back together (i made it easier so I could set it up faster for my tests)

**Comment by Penwin0** [comment]
Screenshot_375
These maints look more like brig cells, its just a rectangle with a bunch of quare rooms

**Comment by LastCrusader105** [comment]
> Screenshot_375
> These maints look more like brig cells, its just a rectangle with a bunch of quare rooms

are you commenting on fact they are empty or on its shape
if its empty, its random rooms so will magic up rooms
if its its shape, I'm open to changing it but quite often buildings are rectangular good example would be offices, do you have any ideas as to how I could change it?

**Comment by LastCrusader105** [comment]
yes travis my boi

**Comment by LastCrusader105** [comment]
changed the floor in crew quarters and sorted patients rooms floor
issue
issue
fixed issue with the floor in plumbing room
maybe fixed morgues issue
issue

**Comment by MaltVinegar** [comment]
They want to get rid of the blessed evac crab 😔

**Comment by triplezeta** [comment]
save the evac crab :crab:

**Comment by LastCrusader105** [comment]
#save Lord Pincalot III

**Comment by Fronsis** [comment]
First of all : Save the crab. at least surround him by glass reee

Remember that the chef and chaplain have morgue access! so they need to have a proper path, there's one above genetics into the watch room of surgery but it's blocked by a wall, having a door there would lead to morgue

**Comment by LastCrusader105** [comment]
> First of all : Save the crab. at least surround him by glass reee
> 
> Remember that the chef and chaplain have morgue access! so they need to have a proper path, there's one above genetics into the watch room of surgery but it's blocked by a wall, having a door there would lead to morgue

issue

crusader why does RD have window into maint? cause hes paranoid, plus it used to be a corridor and its pretty much RDs only weakness
also fixed camera facing wrong way
@BeloneX what does a limb regrower room even have in it?

redid patient room
issue
Prince Pincer IV has his own little garden, you may now seek wisdom from him
issue

**Comment by PowerfulBacon** [comment]
Are there any power wires, they aren't showing up on diffs?

**Comment by LastCrusader105** [comment]
@PowerfulBacon yes, have you not seen them in all image I’ve posted?

**Comment by Penwin0** [comment]
@LastCrusader105 Maint is supposed to be a series of winding halls you could lose someone in, it seems too organized if that makes sense

**Comment by LastCrusader105** [comment]
I will come up with some mock designs

**Comment by LastCrusader105** [comment]
@Penwin0 
issue

**Comment by LastCrusader105** [comment]
all of my maint will have to be redone really, some stats for you
zulu currently has 22 3x3 rooms, delta has 14 box has 14
3x5 rooms Z:7 B:3 D:7
5x3 rooms Z:5 B:4 D:10
5x4 rooms Z:6 B:8 D:10
10x5 Z:4 B:1 D:2
10x10 Z:3 B:1 D:2
so basically too many 3x3 rooms rest are pretty acceptable

issue
issue

File: code/modules/atmospherics/machinery/airalarm.dm (hunk 1 above)
Comment (francinum) [reply code/modules/atmospherics/machinery/airalarm.dm]: This entire file seems to have a lot of dirty changes.

**Review by francinum — CHANGES_REQUESTED** [review]
Figure out your dirty changes on airalarm.dm

You might not have changed them, but they're different.

**Review by LastCrusader105 — COMMENTED** [review]
changing the review for a update
issue
to
issue

**Review by LastCrusader105 — COMMENTED** [review]
ive no idea what im doing with the review stuff, but i have changed what was asked

**Comment by LastCrusader105** [comment]
whole map (minus areas)
zulu

**Comment by qwertyquerty** [comment]
How much work do you estimate you have left?

**Comment by LastCrusader105** [comment]
> How much work do you estimate you have left?

till its perfect? it will never be perfect
right now it all works, apcs, mail, disposals, pipes, just few things like some bits are missing lights (hopefully fixed)
and I have a few ideas to add maints to near sec and near the solars bottom left
I can do it in a day or so
@qwertyquerty

**Comment by PowerfulBacon** [comment]
I still think some of the decals are very, very strange. There are rooms with old kilo level detail (mostly the areas that have been taken from other maps) which is fine, but then there are rooms that have absolutely no decals at all.
Some of the decals are layed out really weirdly too, with rooms being filled with only the top left corner decal on every tile which just looks somewhat strange.

**Comment by LastCrusader105** [comment]
> I still think some of the decals are very, very strange. There are rooms with old kilo level detail (mostly the areas that have been taken from other maps) which is fine, but then there are rooms that have absolutely no decals at all.
> Some of the decals are layed out really weirdly too, with rooms being filled with only the top left corner decal on every tile which just looks somewhat strange.

@PowerfulBacon that is true the decals was mainly for my own memory so I could go back and finish them and also made it look better when I zoomed map out
But I have beefed up Maint, made it more random
And other than few decals the map should be nearly bug free

**Comment by Crossedfall** [comment]
Closing/reopening to force our new checks

**Comment by LastCrusader105** [comment]
ok I will be gathering feedback, summing them up, numbering them and then using that to make a log of what needs fixing and when its done to add to my daily log
Issue 1: too many bees for beestation- my bad didnt know how many spawned, reduced the bees **fixed**
Issue 2: only one engie entrance+smes in engie plus put sms pumps back: I drawn a rough plan and will do it tomorrow pumps replaced with striaght pipe while i tested server made it faster
issue 3: grav gen- personally i dont think its a issue, meta: sm is right by grav gen, delam and grav either blows up or loses its cables to main, box:its accessible from main hallway- if its still classed as a issue we can discuss a solution
issue 4: ss13 sign covered-agreed that was ugly it was my bad i did firelocks near end, moved the locks **fixed**
issue 5: armoury not really a issue but-got tomorrow off so will have a play around then
issue 6: shuttle crash looks silly- moved it all along one so looks like its more impactful **needs review**
issue 7: better transition- made the tiles white and brown rather than grey and dark grey **needs review**
issue 8: too short HOP line-og was gunna be box sized map, extended the hop line i quite like the new one **fixed**
issue 9: bar too cluttered- moved some tables and it seems more open **needs review**
issue 10:light floating- its a wizards light its meant to float, jokes deleted **fixed**
issue 11:med dark- adding more lights testing tomorrow
issue 12: maint dark- same as 11
issue 13:apc not connected-the connecter was wrong way round **fixed**
issue 14: sec maint: true door deleted, comment about cant space dead people, but now you can watch them die and invite others too, could remove **needs review/discussion**
issue 15: maint boxy- he right doe, imma try to play with it more
issue 16: tech storage in maint- im under same opinion as Halinder its always being broken into, hell on box its not even behind reinforced walls **needs discussion**
issue 17:no spot for tesla- i mean i could is that something people be interested in?
issue 18: morgue hard to access and half floored- yes you wanna grave rob you got to walk though scary hall, its meant to look like once it was a normal hallway in use **needs discussion but im happy**
issue 19: chem gangbanging the med corridor wall- hes ight i may put them at diagnal ends ,but rarely does it depressurize outside the room on box and its pressed against wall
issue 20: QM has hardsuit?: yeah apparently
issue 21: camera placement- does need testing all my ai mains i consulted with are banned
issue 22: crematorium/chapel: its now behind reinforced windows, its there so crew can see their loved ones burn, and priests access to interrogation room is to hear his final confession
issue 23: botany is a weird one, people love it people hate it **needs discussion**
issue 24: med checkpoint- removed the vending and put a wall mounted on, removed dr lockers and replaced with a defib **fixed**
issue 25: shuttles- shuttles are future proofing the station when someone decided to add away missions for departments, bit like science **fixed**
issue 26: ttcoms janky- how does it look jank? the ttcoms storage is connected to ttcoms **needs discussion**
issue 27: departments too big- we will see
issue 28: clowns- honk hulk was my doing other was a "patient  spawner" i assumed it would be a guy missing a leg or whatever removed **fixed**
issue 29: sec locker- true fuck knows how it got there **fixed**
issue 30:hallway too small-true moved bar back one, when you said south primary i assumed you meant evac corridor **fixed**
issue 31: one way access disabler?-what? only one way in and out yes
issue 32: damn yous right will be added tomorrow
issue 33:cmo wall-the wall is a weakness on purpose **needs discussion**
issue 34:copy paste- some was definitely exact copies but non was copied and pasted it was all copied by hand (my pipes are layer 1+3) and sometimes it just works example sec is same on every map near enough
issue 35:move engie solar-was OG there but @TeomanTheGreat though better moved so delam doesnt blow it
issue 36: chem vendor- theres what 4 i think, one facing medbay hallway one facing medbay storage one in the entrance to stop randoms hanging about **discuss**
issue 37: toxins- tue but if you look its got a buffer of random rooms **discuss**
issue 38:surgery room- well one not many watch, they dont want people chilling in medbay, the hallway was once used alot now its unused **discuss**
issue 39: chem needs shower- see this is why i like feedback, thats a great idea i will add it when i redo chem
issue 40: wall **fixed**
issue 41: shuttle wont work- not a shuttle just decoration to show how the heads live
issue 42: engie chekpoint- that is good ideas **fixed**

i will address all your positive comments now
AI sat is kilos apparently its the best ai
thank you i liked the idea of soon as you arrive you enter the service section, bit like a airport
more atmos suits cause its bigger
thank you i thought gives engie/ atmos something to do
i thought for rp bigger sec checkpoints would help
mining needs somewhere to chill
when i first started playing i could never find it, meant to be a starter role, noobs shouldnt be murdered in maint tryna find it
thats a thicc ass bridge, now all heads can run department from bridge, hopefully
nanites always confused me, you got to beg them to come to nanite chamber (which on delta is quite deep in science) i will add a door for science to enter nanites
thank you Halinder i tried not to rock the boat like kilo and didnt want to just make big box

the rage cage still needs to remove windows to make it a true rage cage
the ship can be used as a hideout for anyone who wants to chill in relative peace, has all stuff to give it atmos again
thank you

Cheers pill plus side it should keep tide from ruining your round while your trying to do whatever

big thank you for storm for convincing me Endo can be saved, i also like my medbay
and thank you to everyone else for your comments
oh better @francinum as mostly his comments

**Comment by francinum** [comment]
I'm referring to one-way access helpers, like commonly used on medbay doors

If someone doesn't have maint access this door is impassable to them
https://file.house/koVf.png

**Comment by B4CKU** [comment]
personally i'm not a fan of the upload in the ai sat

**Comment by francinum** [comment]
> personally i'm not a fan of the upload in the ai sat

Agreed.

It looks extremely tacked on.

**Comment by LastCrusader105** [comment]
the door is done this is why i need gallus he sees thing i dont

and your all free to join the discord to discuss with me and hash stuff out message me LastCrusader#7452 for a invite, i will still be responding on here and readinfg forums etc but its another way to get hold of me and have your say

**Comment by ghost** [comment]
Watched this map develop from when Crusader first asked around some places about mapping. I'd say by far it looks *a lot* less rougher than I remember and/or was expecting. like It definitely looks odd, but I think it's going places.

**Comment by LastCrusader105** [comment]
what we thinking about from this
image
to
image
please excuse fact it isnt all turned round etc its more a proof of concept 
plus side the super secure engineering storage is now in main corridor
incinerator is now near atmos
And makes engies maint less engie part 2
**relates to issue 3 and 16 also 44**

**Comment by LastCrusader105** [comment]
issue 43: ai upload looks slapped on- tbh thats quite easy fix issue by taking one room off it looks alot better and if i gave it the penis helmet that the top of ai has it might even look better

image


image

**Comment by LastCrusader105** [comment]
issue 19 sorted 
Zulu

**Comment by LastCrusader105** [comment]
dont alot of moving shit around, and expanding a few corridors etc etc
images will be later

**Comment by LastCrusader105** [comment]
added cyro to medbay so ssd can go away 
image

**Comment by LastCrusader105** [comment]
Work continues, after having my map reviewed by a fellow mapper he pointed out 50 things wrong with Endo (few are mout cause he’s not from beestation)

Here are the 50 points
```

1. Aft solars missing one floor
2. birdboat is in a slightly boring area. I'd move him somewhere cooler
3. carpeted hallways look odd in dorms, though this is mostly just my opinion
4. hallway through library isn't great, but isn't 100% necesarry to change
5. wood floors under the windows in the library, change to plating please
6.AI upload. I'd just move it into the station somewhere or entirely redesign the bottom half of the AI sat to make it fit in better
7. having a spaced room in the AI sat makes it easier to break into as a nukie but harder to access legitimately. I'd get rid of that personally, though it's an interesting idea.
8. the shuttle connected to the service hallway doesn't line up with the doors
9. Botany is a bit meh. I'd rearrange the trays to be a bit more portioned and accessible, not just in rows.
10. no easy access between botany and kitchen. Maybe make a delivery chute with disposals between them? 
11. decals aren't symmetrical in the botany office
12. The dorms main room is likely a huge chokepoint. Anyone who enters the station has to walk through there. A segmented dorms is fine but it really needs an actual hallway without tables in the middle.
12.1 you could also make another hallway to the south that goes around all of service, but that's just an idea.
13. the shower room kinda needs the showroom or white floor. It looks odd with grey floors.
14. roundstart rage cage isn't really to my taste. You could make this the stage area and it would be very easily convertable to a rage cage, though it's really up to you.
15. The theater carpeting is weird. This is probably what you intended but I think it's a bit too weird.
16. wall pipes near the theater, next to the maint entrance.
17. The theater is a bit bland and almost unnecesarry due to it not being public access. maybe combine it with the mime/clown room to use that other space for something else
18.emitter in maints is eh.. maybe make it a spawner?
20. The shuttle ruins are pretty cool. I like it as just background stuff, though it might make it hard for nukies to land
21. the maint around aft solars is pretty awkward. I'd remake that to be less spindly or more interesting.
22. cargo is a bit too large in areas. You could repurpose some of that area.
23. the evac med area is a bit odd. I'd remove that
24. The garden is boring. Add some flowers at least and I wouldn't lock away the animals if that's all the room is going to be
25. chem lab seems smallish and the tables make the room smaller, move them to the side or remove them
26. That is a GENIUS use of no-slip floors in virology. I might steal this (not a criticism just a great idea)
27. Same viro as delta? I'd make your own 
28. The medbay shuttle confuses me
29. Two treatment rooms ?
30. The medbay reception allows people to sneak in by hopping two tables. Throw an access-locked windoor on the inner table and that's fixed
31. morgue is a bit tight. I'd move the 3rd row over one and flip them all, so you have a wider hallway between trays and don't lose any
32. robotics is HUGE. I don't know how that space would be better used though
33. limb grower roundstart seems like a bad idea
34. RD's office has a window that's against a wall so it's pointless
35. replace the vents in the server room and xenobio killroom with a single passive vent. It still cools the air and has less atmos calculations
36. the public autolathe in research seems like a bad idea to me, I'd move it behind access at least.
37. the danger lines in the toxins room are weird, maybe change them a bit
38. Remove the roundstart stun baton, or at least make it a spawner near the toxin's room
39. science shuttle?
40. The area with the cremator is missing a directional window
41. tcomms is too exposed. It really needs to be inside the station somewhere
42. the maint near the chapel is really odd. Not really sure what else to say about that.
43. the perma is too small, but I don't know if bee has roundstart prisoners so that might be fine.
44. diagonal walls near cell one, which is no good.
45. random disposal pipes near the detective, under tables and the chair.
46. armory is too big and two secways is too many
47. The teleporter room and tech storage are very cramped. I'd move the teleporter room elsewhere and expand tech storage
48. not sure if this is intentional, but the turbine room is public. You want to put that behind an airlock.
49. The small hallway to the engineering sec cell is awful, I reccomend removing it and just keeping the only entrance inside the sec office. 
50. engineering shuttle???
51. Gravity gen is too close to the SM. if it blows, there will be no gravity for the rest of the round which really sucks.
52. Cut down the smes at the sm to 3, 5 is too many
```

Im working though then asap

**Comment by LastCrusader105** [comment]
All that’s left is the tcoms then all 53 issues are sorted we got on the Ai sat
0F65EB7D-DC09-416C-9A71-B72EBB4A3724

Or by engineering ( a rough sketch)
B440387A-F38D-45B7-B85E-5C54ADA20253

@francinum thoughts?

**Comment by LastCrusader105** [comment]
i have a gift for @BeloneX 
no more crying because your instrument is deep fried 
image

**Comment by LastCrusader105** [comment]
here it is, about as far as I can go solo, theres no pipe issues, bound to be a few door access issues and or camera issues/ lighting issues, but ive gone round multiple times and had a look
maps to swoll to post a whole image so lets do it department by department
heres arrivals, basicly boxes but gets job done and noone hangs around there
Zulu
leaving arrivals you go to the library
Zulu
into the dorms
Zulu
into the service area
Zulu
then to the main corridor which remains mostly unchanged
few things to point out thats a bit unusual round there
the shop now has a extended back room so they can do plumbing or whatnot
Zulu
new viro and plumbing room 
Zulu
our cock and swollen ball ai sat
Zulu
rejigged enginerring 
Zulu
and a brand new shuttle (not exciting)
Zulu


few things I have added that wont be till my bug fixes start are
a maint door leading from sec interrogation, a door from main corridor into nanite area
and lights in clowns music room

I hope you all enjoy playing on Endostation and I cant wait to hear your feedback; i will be looking at Github and the forums monitoring what you find etc
thank you to Beestation for the last year and a half of good times and please allow me to give back to the community, from whom I have taken so much
I look forward to the day Beestation may one day forgive me

Endo

**Comment by Crossedfall** [comment]
Looking a lot better. 

Just a few things I've noticed while skimming the updated map image. There's a weirdly placed holopad in the emitter room that should be moved
image

The pathing for your engine intake is still pretty noodly. In the below I mocked out some other possible paths as an example (I'm sure there are better alternatives as well). There are also a couple of walls that aren't properly reinforced, I marked those as well.
image

You might want to do another pass on areas that have reinforced walls. There are a couple of cases where you have most of the area walled off, but then one or two spots are missing. Functionally these areas appear to be fine, but aesthetically it looks weird (at least to me).
image
image
image
image

**Comment by LastCrusader105** [comment]
in strong they dont look that ugly but fixed all pointed out issues will look for more, but after a while you go blind to the issues (hulk clown being good example)
image
I went with the straight arrow for pipe
buts its 2am here and im done for the day

File: config/maps.txt [hunk 4]
```diff
@@ -17,6 +17,10 @@ map boxstation
 	votable
 endmap
 
+map endostation
```
Comment (Crossedfall) [inline config/maps.txt:20]: Missing the equivalent sage config

Comment (LastCrusader105) [reply config/maps.txt:20]: I assume you have corrected this?
oh i get you sage has its own one
on it now

**Comment by LastCrusader105** [comment]
@Crossedfall thy will is done

**Comment by LastCrusader105** [comment]
the main issue was i got indestructable floors from somewhere (id read not to use them so copied other maps floor tile)
instead of changing var to non destructive i deleted them all, meaning i then had to go find them, between trying to fix the orther issues, this is why after endostation is done i will work on getting beestation maps up to a acceptable standard!

regardless we keep moving on and hammering away at these issues

this latest fix was after 3hs of test playing on Oasis station and I could roam around with more eyes and see

**Comment by LastCrusader105** [comment]
we have a better shuttle (minor apperance issue with a window but its ready for my next PR)
Zulu
various fixes and i sorted power issue by removing 2 smes and replacing another one with a full one solving the roundstart power issues

im grateful for all the feedback and the opportunity to make a map for bee, I apologize that it went so terrible, structurally this should be sound, only issues left MAY be door access issues or decals....hopefully

**Comment by Crossedfall** [comment]
Closing since this user has been banned and will be unable to make any added changes.

**Pull request closed without merging**